src/components/modals/export-data-modal.js

```diff
@@ -126,20 +126,22 @@ const ExportDataModalFactory = () => {
               </div>
               <div className="selection">
                 <StyledFilteredOption
+                  className="unfiltered-option"
                   selected={!filtered}
                   onClick={() => onChangeExportFiltered(false)}
                 >
-                  <div className="filtered-title">Unfiltered Data</div>
-                  <div className="filtered-subtitle">
+                  <div className="filter-option-title">Unfiltered Data</div>
```

Comment:
is there a reason why we changed the className here? It might create issues for people who are using global css selector to override default style of thses dom element

Reply:
I changed to have a more meaningful name